blog/urls.py

```diff
@@ -0,0 +1,8 @@
+"""URLs for the blog app."""
+from django.urls import re_path
+from . import views
+
+
+urlpatterns = [
+    re_path(r'^$', views.post_list, name='post_list'),
```

Comment:
This object does not exist yet in the current commit. It's bad to have "broken" commits in repo. You should aim to have "working" project in all commits of the repo.

Reply:
Sorry, @webknjaz  But I thought 1 page from tutorial it one branch and PR here.

Reply:
Okay, then it should be merged after the other PR with the mentioned view function available.

Reply:
Could you please also add another PR integrating pyflakes into the pre-commit tool first?